code/modules/power/reactor/fuel_rods.dm

```diff
@@ -0,0 +1,246 @@
+/obj/item/fuel_rod
+	name = "uranium-235 fuel rod"
+	desc = "A titanium sheathed rod containing a measure of enriched uranium-dioxide powder inside, and a breeding blanket of uranium-238 around it, used to kick off a fission reaction and breed plutonium fuel respectivly."
+	icon = 'icons/obj/control_rod.dmi'
+	icon_state = "irradiated"
+	w_class = WEIGHT_CLASS_BULKY
+	var/depletion = 0 //Each fuel rod will deplete in around 30 minutes.
+	var/fuel_power = 0.10
+	var/rad_strength = 500
+	var/half_life = 2000 // how many depletion ticks are needed to half the fuel_power (1 tick = 1 second)
+	var/time_created = 0
+	var/og_fuel_power = 0.20 //the original fuel power value
```

Comment:
Any particular reason that fuel_power and og_fuel_power are different?

Reply:
Yes, fuel_power decays over time and og_fuel_power is used to calculate it

Reply:
Actually it could probably be done a better way than this, I'll get to that soon.

Reply:
initial()

> Actually it could probably be done a better way than this, I'll get to that soon.

Reply:
yes